packages/fuels-test-helpers/src/lib.rs

```diff
@@ -5,36 +5,26 @@ use fuel_core::{
     model::{Coin, CoinStatus},
     service::{Config, DbType, FuelService},
 };
-use fuel_crypto::{PublicKey, SecretKey};
 use fuel_gql_client::client::FuelClient;
 use fuel_tx::{Address, Bytes32, UtxoId};
 use rand::Fill;
 use std::net::SocketAddr;
 
 #[cfg(feature = "fuels-signers")]
 mod signers;
+mod wallets_config;
+
 #[cfg(feature = "fuels-signers")]
 pub use signers::*;
+pub use wallets_config::*;
 
-// This constant is used to set a initial balance on wallets
-// mainly used on tests
-pub const DEFAULT_INITIAL_BALANCE: u64 = 1_000_000_000;
-
-pub fn setup_address_and_coins(
-    num_of_coins: usize,
-    amount: u64,
-) -> (SecretKey, Vec<(UtxoId, Coin)>) {
+pub fn setup_coins(owner: Address, num_coins: u64, amount: u64) -> Vec<(UtxoId, Coin)> {
```

Comment:
IMO #264 was more about changing the output of the initial `setup_address_and_coin` than creating a function that only creates the coins.

Reply:
Maybe I got carried away in the process, but still I don't see the benefit of having those two functionalities bundled in a single call, other than saving a line of code. 
This allows for creating UTXOs of varying sizes without the side effect of creating a wallet each time.